lib/config.ts

```diff
@@ -150,6 +150,36 @@ export function get(name: string): any
                 default: 50,
                 arg: 'throttle-rate'
             }
+        },
+        'websocket': {
+            'socket-io': {
+                'enable': {
+                    doc: 'Boolean option to control whether to run socket.io server',
+                    format: Boolean,
+                    default: true,
+                    arg: 'webSocket-socket-io-enable'
```

Comment:
My apologies for not being clear in my comment earlier.  These 4 `arg:` property values can start with `websocket-*` as well since they are CLI arguments.